sycl/source/detail/pi.cpp

```diff
@@ -268,7 +227,8 @@ bool bindPlugin(void *Library, PiPlugin *PluginInformation) {
 }
 
 bool trace(TraceLevel Level) {
-  auto TraceLevelMask = Config<int, SYCL_PI_TRACE>::get();
+  const char *Env = std::getenv(SYCL_PI_TRACE);
```

Comment:
Both getenv & atoi are going to execute on each invocation of trace(). Please get back the one-time Config reader

Reply:
Okay, returned back